src/main/java/sirius/kernel/xml/XMLStructuredOutput.java

```diff
@@ -297,6 +297,28 @@ public StructuredOutput namespacedProperty(@Nonnull String namespace,
         return property(namespace + ":" + name, data, attributes);
     }
 
+    /**
+     * Convenience method for {@link #property(String, Object, Attribute...)} prepending a namespace.
```

Comment:
```suggestion
     * Provides a convenient way for {@link #property(String, Object, Attribute...)} with a prepended namespace.
```

Reply:
done for all copy&paste convenient texts